strings/strings.c

```diff
@@ -9,8 +10,14 @@
 */
 int string_length(char s[])
 {
+    int counter = 0;
+    for (int i = 0; s[i] != '\0'; i++) 
```

Comment:
Good syntax, I like how you used a != conditional in here, more creative than "less than" or "greater than"